components/Header.tsx

```diff
@@ -1,7 +1,35 @@
+import Link from 'next/link';
+import router from 'next/router';
+
 export default function Header() {
   return (
-    <header className="fixed left-0 right-0 top-0 h-12 w-full p-2 flex gap-2 items-center bg-slate-800 text-slate-100">
+    <header className="fixed left-0 right-0 top-0 h-12 w-full px-8 flex gap-2 justify-between items-center bg-slate-800 text-slate-100">
       <h1>GraphiQL</h1>
+      <div className="flex gap-8">
+        <Link href={'/'} className={router.pathname == '/' ? 'underline underline-offset-8' : ''}>
```

Comment:
Maybe create a separate <NavLink> component to avoid repeating logic.

Reply:
done